transcom/mymove
# MAIN - B-21905 - Rename NTS shipment type

## [B-21905](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-21905)

INT PR: https://github.com/transcom/mymove/pull/14419

## Summary

This is to change current NTS shipment type, `HHG_INTO_NTS_DOMESTIC` to `HHG_INTO_NTS` since we will be using these for international shipments. This change applies to everywhere in the app including the enum in the DB.

### How to test

1. Make sure your current NTS shipment still works after running migration on this branch if you had any.
2. Make sure you can see the enum changed to `HHG_INTO_NTS` in the DB for `shipment_type` in `mto_shipments` table.
3. Make sure you can create and go through the normal flow with new or existing NTS shipment.
4. Go to prime sim -> find NTS shipment -> and verify that you see `HHG_INTO_NTS` as the shipment type in prime GET api.
5. Make sure nothing DB related is breaking  🤣 


## Screenshots
![image](https://github.com/user-attachments/assets/8f5536dc-f852-468e-aac6-7e4fff5ab878)
![image](https://github.com/user-attachments/assets/c32b75d0-1443-4ef3-9f7b-5a39a367ca30)
